src/pybind/mgr/rbd_support/module.py

```diff
@@ -188,6 +226,10 @@ def handle_command(self, inbuf, cmd):
                     ex, traceback.format_exc()))
                 raise
 
+        except (rados.ConnectionShutdown, rbd.ConnectionShutdown) as ex:
+            self.log.debug("with_latest_osdmap: client blocklisted")
```

Comment:
`with_latest_osdmap()` method not in pacific:
```suggestion
            self.log.debug("handle_command: client blocklisted")
```

Reply:
Good catch!